core/server/data/migrations/versions/4.43/2022-04-08-10-45-add-subscription-id-to-mrr-events.js

```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('members_paid_subscription_events', 'subscription_id', {
+    type: 'string',
+    maxlength: 24,
+    nullable: true
+});
```

Comment:
Is this intentionally missing a FK to the subscription?

Reply:
Yes - the FK was failing to add because the of the null values for the column.

The plan is to add a migration in the future to populate all missing values, at which point we can add a `NOT NULL` and `REFERENCES` constraint